IPython/core/tests/test_interactiveshell.py

```diff
@@ -143,26 +143,13 @@ def __repr__(self):
 
     def test_future_flags(self):
         """Check that future flags are used for parsing code (gh-777)"""
```

Comment:
Future flags are still a thing, and in Python 3.5 & 3.6 there's a real future flag that makes a difference again (`from __future__ import generator_stop`). So the test does make sense, but there isn't a great way to test it across all supported Pythons. The `barry_as_FLUFL` feature works at least for CPython (I haven't checked if other implementations support it).